code/modules/projectiles/projectile/beams.dm

```diff
@@ -94,6 +94,16 @@
 	name = "laser pellet"
 	icon_state = "scatterlaser"
 	damage = 5
+	wound_bonus = 5
+	bare_wound_bonus = 5
+	damage_falloff_tile = -0.45
+	wound_falloff_tile = -2.5
+
+/obj/projectile/beam/scatter/pathetic
```

Comment:
should probably have a different icon and if possible a unique sound cue (just like a trumpet noise or fart sound)

Reply:
I was thinking the same thing.

Reply:
Resolved this for now